test/entice/logic/vitals_test.exs

```diff
@@ -0,0 +1,36 @@
+defmodule Entice.Logic.VitalsTest do
+  use ExUnit.Case, async: true
+  use Entice.Logic.Attributes
+  alias Entice.Entity
+  alias Entice.Logic.Player
+  alias Entice.Logic.Vitals
+
+  setup do
+    {:ok, e1, _pid} = Entity.start
+    {:ok, e2, _pid} = Entity.start
+
+    Vitals.register(e1)
```

Comment:
you need to add a level before the vitals. these two lines need to go below current line 16 ;)
(since your implementation crashes when there is no level)